Corgibytes.Freshli.Cli/CommandRunners/Cache/DestroyCommandRunner.cs

```diff
@@ -19,14 +20,14 @@ public CacheDestroyCommandRunner(IServiceProvider serviceProvider, Runner runner
     public override int Run(CacheDestroyCommandOptions options, InvocationContext context)
     {
         // Unless the --force flag is passed, prompt the user whether they want to destroy the cache
-        if (!options.Force && !Confirm($"Do you want to completely DELETE the directory {options.CacheDir.FullName}?", context))
+        if (!options.Force && !Confirm($"{CliOutput.CacheDestroyCommandRunner_Run_Prompt} {options.CacheDir.FullName}?", context))
```

Comment:
Any of the translation strings that reference a file name should be adjusted so that the placeholder for the value is included in the translated string. This gives translators the flexibility to place words after the value if it is required. But the `{options.CacheDir.FullName}` construct isn't going to work. Instead, we'll need to use [`String.Format`](https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=net-6.0#system-string-format(system-string-system-object)) and specify the placeholder by specifying `{0}` or `{1}`, for example.